src/Frontend.hs

```diff
@@ -262,7 +258,6 @@ aulaTesting =
   :<|> batchCreateUsers
   :<|> (PageShow <$> Action.persistent mkRandomPassword)
 
-
```

Comment:
@andorp: heh, yes, but I'd need to teach it to tell "real" "haddock" section marks from just random long lines of minus signs, etc. :)